app/api/business/brief_overview_business.py

```diff
@@ -74,6 +79,53 @@ def close_opportunity_early(current_user, brief_id):
     return brief
 
 
+def withdraw_opportunity(current_user, brief_id, data):
```

Comment:
Perhaps the `data` dict argument can be replaced with a more simple `reason` or similarly named string?

Reply:
I've made this update as suggested and raise a `ValidationError` if reason hasn't been provided.